src/main/resources/assets/seedqueue/lang/he_il.json

```diff
@@ -0,0 +1,122 @@
+{
+  "seedqueue.menu.clearing": "מנקה... ",
```

Comment:
Could you mention that it's SeedQueue that's being cleared, I want it to be very clear to the user what mod is showing a screen